modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -535,6 +535,16 @@ class StreamingInput final: public cv::gimpl::GIslandExecutable::IInput
             // Stop case
             return cv::gimpl::StreamMsg{cv::gimpl::EndOfStream{}};
         }
+        // Wrap all input cv::Mats with RMats
+        for (auto& arg : isl_input_args) {
+            if (arg.index() == cv::GRunArg::index_of<cv::Mat>()) {
+                // FIXME: This whole construct is ugly, from
+                // its writing to a need in this in general
```

Comment:
Not sure if this comment is well-written

Reply:
That was your comment :) Will remove

Reply:
LOL now I get it. :)